src/schema/tye-schema.json

```diff
@@ -45,6 +45,10 @@
                 "$ref": "#/definitions/extension"
             }
         },
+        "solution": {
+            "description": "Indicates the solution to use when building project-based services in watch mode. If omitted, those services will be built individually. Specifying the solution can help reduce repeated builds of shared libraries when in watch mode.",
```

Comment:
We use the method `SolutionFIle.Parse(solutionPath)` for processing this property. This can process both `.sln` and Solution Filter (`.slnf`). This should be indicated here in the schema.

Reply:
Updated text.